projects/cashmere/src/lib/checkbox/checkbox.component.scss

```diff
@@ -24,6 +24,10 @@
     }
 }
 
+.hc-children-group {
```

Comment:
@CameronAFaust - since view encapsulation is off, we have to be really specific with our css naming to ensure there isn't any accidental naming collision (View Encapsulation fyi means that css classes apply globally - they aren't just limited to the scope of the component).  So let's change this naming to `hc-checkbox-children-group`